.gitignore

```diff
@@ -22,3 +22,6 @@ locales/
 /PROFILE
 /profile_stats
 /profile_stats.prof
+
+# installed as a submodule
+pyembroidery/
```

Comment:
Was unsure if this is how I'm supposed to get the folder to disappear from my git status. (it still says `modified:   pyembroidery (untracked content)` and I'm not sure why)

Reply:
Do `git submodule update --init`.  This shouldn't need to be in .gitignore.